pkgs/tools/system/gnome-resources/default.nix

```diff
@@ -0,0 +1,77 @@
+{ pkgs
+, fetchFromGitHub
+, stdenv
+, lib
+, appstream-glib
+, cargo
+, desktop-file-utils
+, meson
+, pkg-config
+, rustPlatform
+, rustc
+, glib
+, wrapGAppsHook4
+, systemd
+, polkit
+, dmidecode
+, gtk4
+, libadwaita
+, ninja
+}:
+stdenv.mkDerivation rec {
+  pname = "gnome-resources";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "nokyan";
+    repo = "resources";
+    rev = "v${version}";
+    hash = "sha256-faZ6MDOu/y4+DX9ObjPyVkxKwmLffMJZ93Adf/t8nxQ=";
+  };
+
+  cargoDeps = rustPlatform.importCargoLock {
+    lockFile = "${src}/Cargo.lock";
+    outputHashes = {
+      "plotters-cairo-0.4.0" = "sha256-m7ZT5F7WxjZSGQwuytqdMWUgYvcK2UCU/ntJGIJE+UA=";
+    };
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    desktop-file-utils
+    appstream-glib
+    meson
+    ninja
+    rustc
+    cargo
+    rustPlatform.cargoSetupHook
+    wrapGAppsHook4
+  ];
+
+  buildInputs = [
+    glib
+    gtk4
+    libadwaita
+    polkit
+    systemd
+  ];
+
+  wrapperPath = lib.makeBinPath ([
+    dmidecode
+  ]);
+
+  postFixup = ''
+    # Ensure all dependencies are in PATH
+    wrapProgram $out/bin/resources \
+      --prefix PATH : "${wrapperPath}"
```

Comment:
This introduces double wrapper. At least you should add it to the gapps wrapper. I would prefer to patch the path directly. See https://github.com/NixOS/nixpkgs/pull/272899/files#diff-fae1573573a7335447ad23a3de60070fe61024bc57067ccb4a69e64ad1d5d1eeR112 for example.